drivers/sensor/fxos8700/fxos8700_trigger.c

```diff
@@ -364,6 +366,15 @@ int fxos8700_trigger_init(struct device *dev)
 		return -EIO;
 	}
 
+	if (i2c_reg_update_byte(data->i2c, config->i2c_address,
+				FXOS8700_REG_CTRLREG3,
+				BIT(1),
+				config->gpio_flags & GPIO_ACTIVE_LOW
+				? 0 : BIT(1))) {
+		LOG_ERR("Could not configure interrupt polarity");
+		return -EIO;
+	}
+
```

Comment:
We need to document the usage of `GPIO_ACTIVE_LOW` flag better since it's not that obvious. For now I'll copy the comment I provided for another chip that needed to configure its interrupt line:

> In principle the flags are there to encode cause -> effect relationship. This is easy to understand when taking button as an example. Regardless of the way the button is designed and the circuitry surrounding it if we attach GPIO_ACTIVE_LOW to the pin it means: if the button is pressed we expect input signal on the pin to be low. If we attach GPIO_ACTIVE_HIGH to the pin it means: if the button is pressed we expect input signal on the pin to be high. The same holds true for MCP2515 chip. The flag is there to tell the driver what happens when MCP2515 asserts its interrupt line. In a typical case the interrupt output of MCP2515 will be wired directly to the GPIO pin on the MCU. This should be described in DTS by adding GPIO_ACTIVE_LOW flag to the pin property. However, it's also legal to attach GPIO_ACTIVE_HIGH flag in case the signal from MCP2515 to MCU is routed on the board via an inverter.

In short the `GPIO_ACTIVE_LOW`, `GPIO_ACTIVE_HIGH` flags are there to adjust the driver to the board and not the other way round. We don't need to configure interrupt level of the FXOS8700. This code can be removed.

Reply:
Done